metadata-ingestion/src/datahub/ingestion/source/ge_profiling_config.py

```diff
@@ -115,6 +115,10 @@ class GEProfilingConfig(ConfigModel):
         description="Profile tables only if their row count is less then specified count. If set to `null`, no limit on the row count of tables to profile. Supported only in `snowflake` and `BigQuery`",
     )
 
+    profile_table_row_estimate: bool = Field(
+        default=False, description="Whether to get only row count estimates"
```

Comment:
```suggestion
        default=False, description="Whether to get only row count estimates. Only supported for Postgres."
```

Reply:
@hsheth2 Updated it to `profile_table_row_count_estimate` let me know if you have other thoughts